README.md

```diff
@@ -27,6 +27,10 @@ At the same time, a diverse and creative community of tools and services has spr
 
 Imagine a declarative Kubernetes-style API for _anything_, supported by an ecosystem of Kubernetes-aware tooling, separate from Kubernetes-the-container-orchestrator.
 
+But a control plane without any workloads isn't very compelling... We want to uplevel the majority of Kubernetes applications today by allowing them to move and spread across clusters without changing a line of YAML. We believe moving a workload between clusters should be as easy as moving a pod between nodes.
```

Comment:
I would remove the ellipsis here and tweak a bit (after re-reading this about ten times:

> But even a generic control plane is only as good as the use case it enables.  We want to give existing Kubernetes users a new superpower - take your existing applications and allow them to move or spread across clusters without having to change a single line of YAML. We believe moving a workload between clusters should be as easy as moving a pod between nodes.